config/default/manager/manager.yaml

```diff
@@ -34,10 +34,10 @@ spec:
         resources:
           limits:
             cpu: 100m
-            memory: 30Mi
+            memory: 300Mi
```

Comment:
What's going on here? Why do we need more mem?

Reply:
When I deploy to a real cluster I got OOM, I think the cache is bigger when you have more resources on the kubernetes cluster.